### PR TITLE
added validator/nominator flags

### LINKED ISSUE #3064: Display validator/nominator tags in account sidebar

### PR DESCRIPTION
**Pull request by jhonalino**
closes #3064

### CODE REVIEW

**Comment by jacogr** [comment]
Thank you! will take a look when fresh.

File: packages/react-hooks/src/useAccountInfo.ts [hunk 1]
```diff
@@ -38,6 +41,8 @@ export function useAccountInfo (value: string | null, isContract = false): UseAc
   const { isAddress } = useAddresses();
   const accountInfo = useCall<DeriveAccountInfo>(api.derive.accounts.info as any, [value]);
   const accountFlags = useCall<DeriveAccountFlags>(api.derive.accounts.flags as any, [value]);
+  const nominator = useCall<Nominations>(api.query.staking.nominators as any, [value]);
+  const validator = useCall<ValidatorPrefs>(api.query.staking.validators as any, [value]);
```
Comment (jacogr) [inline packages/react-hooks/src/useAccountInfo.ts:45]:
```suggestion
  const nominator = useCall<Nominations>(api.query.staking.nominator, [value]);
  const validator = useCall<ValidatorPrefs>(api.query.staking.validators, [value]);
```

The `as any` in the previous lines should be removed as well (at some point in the past the derives didn't work well/at-all typing-wise with `useCall`, so those were hacks. The `query.*` was always good :)

**Review by jacogr — APPROVED** [review]
Thank you! Sorry for the long delay here.

One small comment, but I've committed it.

File: packages/react-hooks/src/useAccountInfo.ts [hunk 2]
```diff
@@ -38,6 +41,8 @@ export function useAccountInfo (value: string | null, isContract = false): UseAc
   const { isAddress } = useAddresses();
   const accountInfo = useCall<DeriveAccountInfo>(api.derive.accounts.info as any, [value]);
   const accountFlags = useCall<DeriveAccountFlags>(api.derive.accounts.flags as any, [value]);
+  const nominator = useCall<Nominations>(api.query.staking.nominator, [value]);
```
Comment (jacogr) [inline packages/react-hooks/src/useAccountInfo.ts:44]:
```suggestion
  const nominator = useCall<Nominations>(api.query.staking.nominators, [value]);
```

Ooops, type in the previous adjustment. Sorry.

**Comment by jhonalino** [comment]
sweet thank you so much!!! I've found a couple of different places to add my changes but in the end, I've applied the updates on these 4 files.  I'm glad you approved the updates. but even if you don't I look forward to your comments!! 

one curious question what does ValidatorPrefs mean? I tried Validators as a type but it didn't work

**Comment by jacogr** [comment]
It is an on-chain type - it basically (atm) just contains the validator commission. The validators set it when they start validating.

**Pull request merged**